engine/docs/src/script_doc.py

```diff
@@ -434,20 +501,19 @@ def add_group_to_doc_dict(doc_dict):
                 "group": "SCRIPT",
                 "namespaces": ["buffer", "builtins", "html5", "http", "image", "json", "msg", "timer", "vmath", "zlib"]
             },
-            {
-                "group": "EXTENSIONS",
-                "namespaces": ["facebook", "iap", "iac", "push", "webview"]
-            },
-            {
-                "group": "DEFOLD SDK",
-                "namespaces": ["dmAlign", "dmArray", "dmBuffer", "dmConditionVariable", "dmConfigFile", "dmExtension", "dmGraphics", "dmHash", "dmJson", "dmLog", "dmMutex", "dmScript", "sharedlibrary"]
-            },
             {
                 "group": "LUA STANDARD LIBS",
                 "namespaces": ["base", "bit", "coroutine", "debug", "io", "socket", "math", "os", "package", "string", "table"]
             },
         ]
 
+        # Should we get this from the actual @document segment instead, instead of trying to do it dynamically?
+
+        path = info.get('path', '')
+        if 'engine' in path:
+            info["group"] = "DEFOLD SDK"
+            return
```

Comment:
Took a while to realize why the new documentation didn't get grouped correctly :)

Reply:
Ah, yeah, I forgot we divide documentation into groups here in script_doc.py